### DISCUSSION TITLE
How do we get the pictures

### DISCUSSION THREAD

**Discussion by Chocaholics** [zcpua/midjourney-api#181] [Q&A]
Hello! 
I have some questions regarding the getting the pictures.
After sending the initial request, how did you get the target picture?
So for example, if two client sent the same prompt at the same time, will you return the two difference responses from discord or just the same one?
I have this confusion because you are keep GETing the message according to the prompt and the content... right?

**Comment by zcpua** [comment]
nonce

**Comment by Chocaholics** [comment]
oh thanks... I completely ignored this attribute. 
do you mean that this field can be incorporated into a GET request?
